### ISSUE TITLE
Creating answers with keyboards

### ISSUE THREAD

**Issue by jemsz** [guidone/node-red-contrib-chatbot#257]
Hi All,
Love this project so far, some of the apps I have created are working great!
One question I have is regarding the keyboard feature.

1.) I create a keyboard with 4 buttons. However, I can only choose 1 answer for all 4 buttons?
2.) Can we use conditional logic with keyboards? E.g. Choose button A on Keyboard 1 >>> then it goes to another set of buttons from an alternative keyboard (Keyboard 2)?

Any screenshots or tips on the above would be very helpful.
Thanks!

**Comment by guidone** [comment]
Yes you can only press only one button but you can build multi-stage menu with buttons, see this tutorial

https://github.com/guidone/node-red-contrib-chatbot/wiki/Edit-Message-Buttons
